auth/src/main/java/com/firebase/ui/auth/ErrorCodes.java

```diff
@@ -67,6 +73,8 @@ public static String toFriendlyMessage(@Code int code) {
                 return "Developer error";
             case PROVIDER_ERROR:
                 return "Provider error";
+            case ANONYMOUS_UPGRADE_MERGE_CONFLICT:
+                return "Merge conflict";
```

Comment:
This should be a slightly more user friendly message like `User account conflict`

Reply:
:+1: